src/main/drivers/sensor.h

```diff
@@ -34,10 +34,13 @@ typedef enum {
     CW270_DEG_FLIP = 8
 } sensor_align_e;
 
-typedef union busDevice_t {
+typedef union busDevice_u {
```

Comment:
Well, that depends if you want to support runtime or compile time typing of the bus. In any case we can defer that decision until we actually start on changing the I2C code.